developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
 slug: "mint-nft-cli"
 ---
 
-# Mint an NFT
+# Mint an NFT with Aptos CLI
+
+This tutorial lets you use the Aptos CLI to mint non-fungible tokens (NFTs) in Aptos testnet so you can see how the process works and employ related functions in your code.
```

Comment:
I think this is minting NFTs in devnet / testnet / mainnet depending on what network the user chooses when creating their accounts from CLI (I'd imagine most people will use devnet though)

Reply:
Bummer. I used testnet because that's what Workshop #1 used, and we didn't specify a network in the new materials, such as:
https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/1-Create-NFT/sources/create_nft.move

I have updated the introduction to note that although this doc employs testnet, users may choose devnet or testnet assuming funding.

I can do a straight replacement of testnet with devnet but am not sure that will simply work. Your call.

Reply:
I replaced everywhere but in output highlighted below.